src/components/AppSettingsDialog/AppSettingsDialog.vue

```diff
@@ -167,16 +167,18 @@ export default {
 			this.$emit('update:open', false)
 		},
 
-		// Remove selected section once the user starts scrolling
 		handleScroll() {
 			if (!this.linkClicked) {
-				debounce(() => {
-					this.selectedSection = ''
-					document.activeElement.blur()
-				}, 300)
+				this.blur()
 			}
 		},
 
+		// Remove selected section once the user starts scrolling
+		blur: debounce(function() {
+			this.selectedSection = ''
+			document.activeElement.blur()
```

Comment:
So when scrolling and clicking (or tabbing through items) this will unfocus the input after "random" 300ms
Sounds unexpected from a user perspective

Reply:
:ok_hand: